README.md

```diff
@@ -50,6 +50,21 @@ Unpublish your post using:
 
     $ bundle exec jekyll unpublish _posts/2014-01-24-my-new-draft.md
 
+## Configuration
+
+To tune default plugin configuration edit `jekyll_compose` section of jekyll config file.
+
+```
+  jekyll_compose:
+    auto_open: true
+```
+
+and make sure that you have `EDITOR` or `JEKYLL_EDITOR` env variable set.
```

Comment:
Let's say "environment variable" instead of "env variable" here, just to be clear.